@here/harp-mapview/test/UtilsTest.ts

```diff
@@ -168,6 +168,6 @@ describe("map-view#Utils", function() {
             const cube = new THREE.Mesh(geometry, material);
             scene.add(cube);
         }
-        expect(MapViewUtils.estimateObjectSize(scene)).to.be.equal(4078548);
+        expect(MapViewUtils.estimateObjectSize(scene)).to.be.equal(4082552);
```

Comment:
Why do these tests need to change?

Reply:
Because internal representation of 3JS structures changed ...
Just this.
This test measures estimated size of objects in bytes, looks like 3JS added some members here and there.